src/fiber/schema/fiber.mol

```diff
@@ -136,6 +144,59 @@ table ReestablishChannel {
     remote_commitment_number: Uint64,
 }
 
+table AnnouncementSignatures {
+    channel_id: Byte32,
+    channel_outpoint: OutPoint,
+    node_signature: EcdsaSignature,
+    partial_signature: Byte32,
+}
+
+table NodeAnnouncement {
+    // Signature to this message.
+    signature: EcdsaSignature,
+    // Tentatively using 64 bits for features. May change the type later while developing.
+    // https://github.com/lightningdevkit/rust-lightning/blob/8c1b3d1263f6b9247c0c819039ef2027dc4d4588/lightning/src/ln/features.rs#L448-L457
+    // rust-lightning uses a Vec<u8> here.
+    features: Uint64,
+    // Timestamp to the node announcement update, later update should have larger timestamp.
+    timestamp: Uint64,
+    node_id: Pubkey,
+    // Must be a valid utf-8 string of length maximal length 32 bytes.
+    alias: Byte32,
+    // All the reachable addresses. 
+    address: BytesVec,
+}
+
+table ChannelAnnouncement {
+    node_signature_1: EcdsaSignature,
+    node_signature_2: EcdsaSignature,
+    // Signature signed by the funding transaction output public key.
+    ckb_signature: SchnorrSignature,
+    // Tentatively using 64 bits for features. May change the type later while developing.
+    // https://github.com/lightningdevkit/rust-lightning/blob/8c1b3d1263f6b9247c0c819039ef2027dc4d4588/lightning/src/ln/msgs.rs#L1200-L1200
+    // rust-lightning uses a Vec<u8> here.
```

Comment:
add `channel_short_id` and `timestamp`?

Reply:
As far as I can recall, bolts 07 designed `ChannelAnnouncement` message to be immutable. That is to say, we will only post updates to the channel with `ChannelUpdate` message. This is different from `NodeAnnouncement`. So we don't need `timestamp` here. `channel_short_id` was removed because we didn't have a way to return the transaction index and @quake hinted that ckb is prone to reorganize. So using channel outpoint is actually more suitable here.

Reply:
Ok.